docs/src/api/params.md

```diff
@@ -355,16 +369,25 @@ this request body. If this parameter is specified `content-type` header will be
 unless explicitly provided. File values can be passed either as [`fs.ReadStream`](https://nodejs.org/api/fs.html#fs_class_fs_readstream)
 or as file-like object containing file name, mime-type and its content.
 
+## csharp-fetch-option-multipart
+* langs: csharp
+- `multipart` <[FormData]>
+
+Provides an object that will be serialized as html form using `multipart/form-data` encoding and sent as
+this request body. If this parameter is specified `content-type` header will be set to `multipart/form-data`
+unless explicitly provided. File values can be passed either as [`fs.ReadStream`](https://nodejs.org/api/fs.html#fs_class_fs_readstream)
+or as file-like object containing file name, mime-type and its content.
+
 ## js-python-fetch-option-data
-* langs: js, python
-- `data` <[string]|[Buffer]|[Serializable]>
+* langs: js, python, csharp
+- `data` <[string]|[Buffer]|[any]>
```

Comment:
Why not Serializable? Whatever we prefer here, let's keep it consistent with `RequestOptions.setData.data` (update both to use same type).

Reply:
Serializable ends up in .NET to a generic, I can probably revert and massage the generator to prevent that in some cases.